packaging/nuget/packaging.py

```diff
@@ -327,19 +327,19 @@ def build (self, buildtype):
                                               a.info.get('arch'),
                                               a.info.get('bldtype'))
             if 'toolset' not in a.info:
-                a.info['toolset'] = 'v120'
```

Comment:
wondering if this affects OOTB compatibility anywhere important, but 2015 was a long time ago...

Reply:
Yeah, I'm not completely sure either. But we'll find out